app/code/community/PedroTeixeira/Correios/Model/Source/PlpStatus.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * This source file is subject to the MIT License.
+ * It is also available through http://opensource.org/licenses/MIT
+ *
+ * @category  PedroTeixeira
+ * @package   PedroTeixeira_Correios
+ * @author    Pedro Teixeira <hello@pedroteixeira.io>
+ * @copyright 2015 Pedro Teixeira (http://pedroteixeira.io)
+ * @license   http://opensource.org/licenses/MIT MIT
+ * @link      https://github.com/pedro-teixeira/correios
+ */
+class PedroTeixeira_Correios_Model_Source_PlpStatus extends Mage_Eav_Model_Entity_Attribute_Source_Abstract
+{
+    
+    const OPEN = 0;
```

Comment:
Essa classe existe só por essa constante?

Reply:
Sim. Deverá fazer mais sentido na medida que a integração com o SIGEP for mais explorada.
_p.s. Essa integração explora menos de 10% das funções do SIGEP._